### ISSUE TITLE
Could not find MANIFEST for package source

### ISSUE THREAD

**Issue by kthth** [mbutterick/pollen-users#132]
I'm using Debian in WSL (Windows Subsystem for Linux) and installed racket with apt. It is version 7.9 [bc].

When trying to install pollen with `raco pkg install pollen`, I get

```
Resolving "pollen" via https://download.racket-lang.org/releases/7.9/catalog/
Resolving "pollen" via https://pkgs.racket-lang.org
Cloning remote directory https://git.sr.ht/~mbutterick/pollen
raco pkg install: could not find MANIFEST for package source
  source: https://git.sr.ht/~mbutterick/pollen
  possible solution:
   If the URL is intended to refer to a Git repository, use
     https://git.sr.ht/~mbutterick/pollen.git
   so that the URL ends in ".git"
```

How do I get around that?

**Comment by sorawee** [comment]
As I understand, the fix is to register `pollen` on pkgs.racket-lang.org with the `git+https` protocol. E.g., `git+https://git.sr.ht/~mbutterick/pollen`

**Comment by mbutterick** [comment]
Thanks. I didn”t know the `git+https` notation. I’ve updated the package.

**Comment by privong** [comment]
I was having the same issue. It's now working after the fix. Thanks.

**Comment by kthth** [comment]
I now get

```
$ raco pkg install pollen
Resolving "pollen" via https://download.racket-lang.org/releases/7.9/catalog/
Resolving "pollen" via https://pkgs.racket-lang.org
raco pkg install: no such directory
  path: git+https://git.sr.ht/~mbutterick/pollen/
```

Solved it differently now by cloning the source code and installing with the local copy: `raco pkg install --link pollen`. All dependencies were installed without problems.

**Comment by mbutterick** [comment]
@sorawee Where does this `git+https` fix come from?

**Comment by mbutterick** [comment]
I just switched the package-server link back to GitHub. There is a note on the package-server admin page confirming that `git+http` is only available for Racket 8.1+.
